### DISCUSSION TITLE
How Do I Add A Custom Girlfriend

### DISCUSSION THREAD

**Discussion by ghost** [KadeArchive/Kade-Engine#902] [Q&A]
Ok I Tried To Add Zelda Gf To My Mod And For Some Reason It Shows Normal Gf How Do I Do It Have All Png Xml Files In The Right Directory

**Comment by isakube** [comment]
(DISCUSIONNNNNNN)

You need to change files in the source code and then [build the game from the source code.](https://kadedev.github.io/Kade-Engine/building) A character guide is coming soon, but for now, i'm gonna wait for somebody who actually remembers everything. As for now, https://www.youtube.com/watch?v=ac94p0zqqYM&t=3564s

**Comment by ghost** [comment]
ya i have kade engine
